js-core-3/week-3/lesson.md

```diff
@@ -562,9 +563,37 @@ sayHi(); // Hi, Mike
 
 This behaviour will become important in the near future, when we will need to start passing methods to be called other parts of our code.
 
-### Exercise 1
+### Exercises
+
+In break out groups solve the following exercises. Only run code to verify answers. Try to understand what each line of code is doing and the effect it has.
+
+```js
+// Exercise 1
+class Car {
+  goFaster() {
+    this.speed = 100; // what is the value of this code below is run
+  }
+}
+
+const ferrari = new Car();
+ferrari.goFaster();
```

Comment:
Should we add what the expected output of calling this method is?

Reply:
The main thing I wanted to illustrate is how the method is getting called rather than wider functionality.